git-secrets

```diff
@@ -128,8 +165,8 @@ regular_grep() {
 # Takes into account allowed patterns, and if a bad match is found,
 # prints an error message and exits 1.
 process_output() {
-  local status="$1" output="$2"
-  local allowed=$(load_allowed)
+  local status="$1" output="$2" new_rev="$3"
+  local allowed=$(load_allowed $new_rev)
```

Comment:
This is so the server can know about any changes to the repo's .gitallowed file. Otherwise load_allowed will only know about the most recent .gitallowed that is already on the server.